bfdd/bfd.c

```diff
@@ -239,9 +239,6 @@ void bfd_clear_stored_pkt(struct bfd_session *bs)
 	if (bs->bfd_tx_pkt_stored) {
 		memset(&(bs->bfd_tx_pkt), 0, sizeof(struct bfd_pkt));
 		bs->bfd_tx_pkt_stored = false;
-
-		log_debug_info("clear-packet: session-id: %d",
-						bs->discrs.my_discr);
 	}
```

Comment:
can you please aggregate that commit into that one : 'bfdd: Storing Tx packet'
to avoid unnecessary commits, please.